documentation/modules/ROOT/pages/transformations/event-flattening.adoc

```diff
@@ -268,6 +271,24 @@ Specify `none` to keep the change event record in the stream. The record contain
 Specify `rewrite` to keep the change event record in the stream and edit the record to have a `value` field that contains the key/value pairs that were in the `before` field and also add `+__deleted: true+` to the `value`. This is another way to indicate that the record has been deleted. +
  +
 When you  specify `rewrite`, the updated simplified records for `DELETE` operations might be all you need to track deleted records. You can consider accepting the default behavior of dropping the tombstone records that the {prodname} connector creates.
+[NOTE]
+====
+Note: It will be removed in further release, use xref:extract-new-record-state-delete-tombstone-handling-mode[`delete.tombstone.handling.mode`] instead
+====
+
+|[[extract-new-record-state-delete-tombstone-handling-mode]]xref:extract-new-record-state-delete-tombstone-handling-mode[`delete.tombstone.handling.mode`]
+|
+|{prodname} generates a change event record for each `DELETE` operation. It will determine the event flattening SMT how to handle `DELETE` events from the stream.
+
+`drop`:: The SMT removes both the `DELETE` event and `TOMBSTONE` from the stream.
+`tombstone`:: The SMT only keeps the `TOMBSTONE` in the stream. The record contains only `"value": "null"`.
+`rewrite`:: The SMT keeps the change event record in the stream and edit the record to have a `value` field that contains the key/value pairs that were in the `before` field and also add `+__deleted: true+` to the `value` and remove the `TOMBSTONE`.
+This is another way to indicate that the record has been deleted.
+`rewrite-with-tombstone`:: The SMT has the same behavior from `rewrite` except to keep the `TOMBSTONE`.
+[NOTE]
+====
+Note: When specify a value from above will enable this new config in event flatting SMT, it has a higher priority than the deprecated xref:extract-new-record-state-drop-tombstones[`drop.tombstones`] and xref:extract-new-record-state-delete-handling-mode[`delete.handling.mode`].
+====
```

Comment:
```suggestion
`drop`:: The SMT removes both `DELETE` events and `TOMBSTONE`records from the stream.
`tombstone` (default):: The SMT retains `TOMBSTONE` records in the stream.
The `TOMBSTONE` record contains only the following value: `"value": "null"`.
`rewrite`:: The SMT retains the change event record in the stream and makes the following changes:

* Adds a `value` field to the record that contains the key/value pairs from the `before` field of the original record.
* Adds `+__deleted: true+` to the `value` of the record.
* Removes `TOMBSTONE` records.
+
This setting provides another way to indicate that the record has been deleted.

`rewrite-with-tombstone`:: The SMT behaves as it does when you select the `rewrite` option, except that it also retains `TOMBSTONE` records.
```

Reply:
Moved the note to the follow immediately after the first paragraph of the description.